src/nodes.coffee

```diff
@@ -3028,6 +3063,20 @@ UTILITIES =
       return -1;
     }
   "
+  
+  # copy object properties excluding the list of keys
+  extractObjectWithoutKeys: (o) -> "
```

Comment:
CS2 assumes ES2015+. Do we need this function? Is there no ES function like `Object.keys` that can do some or all of this work for us?

I’m not opposed to the function if we need it; just trying to avoid helpers unless we need them.